trento-docs-site/modules/user-guide/nav_user_guide.adoc

```diff
@@ -3,6 +3,10 @@
  ** xref:trento-lifecycle.adoc[Lifecycle]
  ** xref:trento-requirements.adoc[Requirements]
  ** xref:trento-install-server.adoc[Installing Trento Server]
```

Comment:
Let's move the agent installation section right after the server installation section:
```suggestion
 ** xref:trento-install-server.adoc[Installing Trento Server]
 ** xref:trento-install-agents.adoc[Installing Trento Agents]
```